CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to Repositories
```

Comment:
Instead of duplicating the guidelines, I suggest we [add them to `style-guidelines`](https://github.com/pulp-platform/style-guidelines/pull/5) and simply link to that file from here.

Reply:
Thanks